src/mscorlib/model.xml

```diff
@@ -6703,6 +6712,8 @@
       <Member Name="AddRef(System.IntPtr)" />
       <Member Name="AllocHGlobal(System.Int32)" />
       <Member Name="AllocCoTaskMem(System.Int32)" />
+      <Member Name="BindToMoniker(System.String)" />
+      <Member Name="ChangeWrapperHandleStrength(System.Object,System.Boolean)" />
```

Comment:
They are in https://raw.githubusercontent.com/dotnet/standard/master/netstandard/ref/mscorlib.cs